mavros/src/plugins/actuator_control.cpp

```diff
@@ -0,0 +1,109 @@
+/**
+ * @brief ActuatorControl plugin
+ * @file actuator_control.cpp
+ * @author Marcel Stüttgen <stuettgen@fh-aachen.de>
+ *
+ * This plugin will take ros message mavros::ActuatorControl and send it to FCU
+ * via MAVLINK. It can be used for direct servo control in offboard mode
+ *
```

Comment:
Don't need this. We'll put it into the wiki like other plugins. Good to maintain consistency.